src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -185,11 +185,10 @@ import scala.language.implicitConversions
   *
   *  @tparam A    the type of the elements contained in this lazy list.
   *
-  *  @author Martin Odersky, Matthias Zenger
+  *  @author NthPortal
```

Comment:
I think we should leave the name of the original authors, in addition to your name.

Reply:
or just remove the line entirely, the tradition of `@author` annotations is from early LAMP days, it's been a long time since we bothered adding or updating them in this repo